app/src/sharedTest/java/org/oppia/android/app/devoptions/DeveloperOptionsActivityTest.kt

```diff
@@ -248,10 +248,13 @@ class DeveloperOptionsActivityTest {
       )
     ).use {
       testCoroutineDispatchers.runCurrent()
-      assertThrows(RuntimeException::class) {
+      val exception = assertThrows(RuntimeException::class) {
         scrollToPosition(position = 2)
         onView(withId(R.id.force_crash_text_view)).perform(click())
       }
+      assertThat(exception.cause)
+        .hasMessageThat()
+        .contains("Force crash occurred")
```

Comment:
Can this fit on one line? If so, prefer to keep it on one line.

Reply:
Done.